dot/network/service.go

```diff
@@ -669,21 +669,21 @@ func (s *Service) processMessage(msg peerset.Message) {
 			var err error
 			addrInfo, err = s.host.discovery.findPeer(peerID)
 			if err != nil {
-				logger.Debugf("failed to find peer id %s: %s", peerID, err)
+				logger.Errorf("failed to find peer id %s: %s", peerID, err)
 				return
 			}
 		}
 
 		err := s.host.connect(addrInfo)
 		if err != nil {
-			logger.Debugf("failed to open connection for peer %s: %s", peerID, err)
+			logger.Errorf("failed to open connection for peer %s, error: %s", peerID, err)
```

Comment:
To keep it consistent with other messages. Plus we already mention `failed ...` so it's kind of clear this didn't work :wink: 

```suggestion
			logger.Errorf("failed to open connection for peer %s: %s", peerID, err)
```

Reply:
would say this is a warn or debug